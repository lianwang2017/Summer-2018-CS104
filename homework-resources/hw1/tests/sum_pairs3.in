12
104
560
982
254
56
457
19
790
852
334
957
2
