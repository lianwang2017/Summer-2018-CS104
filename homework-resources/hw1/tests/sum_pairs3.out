106
1517
1316
1106
846
476
